invokeai/frontend/src/app/constants.ts

```diff
@@ -52,7 +52,7 @@ export const UPSCALING_LEVELS: Array<{ key: string; value: number }> = [
 
 export const NUMPY_RAND_MIN = 0;
 
-export const NUMPY_RAND_MAX = 4294967295;
+export const NUMPY_RAND_MAX = 9007199254740991;
```

Comment:
not sure about this - cross-browser JS behaviour breaks down after ~28 bits IIRC

Reply:
I don't believe browser differences are relevant with respect to the precision of the Javascript number type.  Javascript numbers are simply 64-bit IEEE 754 floating point values and that isn't going to change browser to browser.  I have set `NUMPY_RAND_MAX` to Number.MAX_SAFE_INTEGER](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/MAX_SAFE_INTEGER)

See also:
https://stackoverflow.com/questions/307179/what-is-javascripts-highest-integer-value-that-a-number-can-go-to-without-losin

Reply:
Can you explicitly reference `Number.MAX_SAFE_INTEGER` here instead of the decimal representation of it? That would make things clearer.

Also, this leaves the `NUMPY_RAND_MAX` name misleading, as it doesn't have anything to do with numpy.